vm/systemSmartContracts/governance.go

```diff
@@ -1010,7 +1034,11 @@ func (g *governanceContract) getTotalStake(validatorAddress []byte) (*big.Int, e
 	return validatorData.TotalStakeValue, nil
 }
 
-func (g *governanceContract) saveUserVotes(address []byte, votedList *OngoingVotedList) error {
+func (g *governanceContract) saveUserVotes(address []byte, votedList *OngoingVotedListV2) error {
+	if !g.enableEpochsHandler.IsFlagEnabled(common.GovernanceFixesFlag) {
```

Comment:
Seems that starting with activation epoch you save in the new structure.
If there is already data in the old struct (Delegated) for the address, how will this user be treated?
I don't see the double vote protection working between versions when user voted before and after the activation epoch for the same proposal through the same contract.